### ISSUE TITLE
feature: fix *some* cases of call of fmt.Sprintf call as msgAndArgs

### ISSUE THREAD

**Issue by dolmen** [Antonboom/testifylint#33]
Most `assert` functions have an `msgAndArgs` arguments variadic list. Its expansion is handled by the internal [`messageFromMsgAndArgs`](https://github.com/stretchr/testify/blob/db8608ed63f5bd9abdd03f669f5bb80569c114b6/assert/assertions.go#L262) function.

Some cases of a call to `fmt.Sprintf` as the value for `msgAndArgs` could be replaced by directly passing the message and arguments to the assert function.

Cases:
*to be defined*

Here is an example in [`github.com/stretchr/testify/mock`](https://github.com/stretchr/testify/blob/db8608ed63f5bd9abdd03f669f5bb80569c114b6/mock/mock_test.go#L1965C2-L1965C2): 
 
```go
assert.Greater(t, elapsedTime, waitMs, fmt.Sprintf("Total elapsed time:%v should be at least greater than %v", elapsedTime, waitMs))
```

**Comment by Antonboom** [comment]
Hi! Thank you for request.

Covered by [CONTRIBUTING.md#no-fmt-mess](https://github.com/Antonboom/testifylint/blob/master/CONTRIBUTING.md#no-fmt-mess)?

**Comment by dolmen** [comment]
When I will have a more precise idea of the cases, I will propose a patch to [CONTRIBUTING.md#no-fmt-mess](https://github.com/Antonboom/testifylint/blob/master/CONTRIBUTING.md#no-fmt-mess).

But for now it's better to close this issue.

**Comment by Antonboom** [comment]
Work in progress.
Blocked by https://go-review.googlesource.com/c/tools/+/580555

**Comment by ccoVeille** [comment]
Initial issue https://github.com/golang/go/issues/66887#issue-2251316432

**Comment by ccoVeille** [comment]
Great !!
